create_pkgbuild.py

```diff
@@ -237,55 +257,59 @@ def is_same_version(self, pkgbuild_file):
-    url='%(site_url)s'
-
-    pkgname='ros-%(distro)s-%(arch_package_name)s'
-    pkgver='%(package_version)s'
-    arch=('any')
-    pkgrel=%(package_release)s
-    license=('%(license)s')
-
-    ros_makedepends=(%(ros_build_dependencies)s)
-    makedepends=('cmake' 'ros-build-tools'
-    ${ros_makedepends[@]}
-    %(other_build_dependencies)s)
-
-    ros_depends=(%(ros_run_dependencies)s)
-    depends=(${ros_depends[@]}
-    %(other_run_dependencies)s)
-
-    build() {
-        # Use ROS environment variables
-        source /usr/share/ros-build-tools/clear-ros-env.sh
-        [ -f /opt/ros/%(distro)s/setup.bash ] && source /opt/ros/%(distro)s/setup.bash
-
-        # Create build directory
-        [ -d ${srcdir}/build ] || mkdir ${srcdir}/build
-        cd ${srcdir}/build
-
-        # Fix Python2/Python3 conflicts
-        /usr/share/ros-build-tools/fix-python-scripts.sh -v %(python_version_major)s ${srcdir}/${_dir}
-
-        # Build project
-        cmake ${srcdir}/${_dir} \\
-                -DCMAKE_BUILD_TYPE=Release \\
-                -DCATKIN_BUILD_BINARY_PACKAGE=%(binary)s \\
-                -DCMAKE_INSTALL_PREFIX=/opt/ros/%(distro)s \\
-                -DPYTHON_EXECUTABLE=%(python_executable)s \\
-                -DPYTHON_INCLUDE_DIR=%(python_include_dir)s \\
-                -DPYTHON_LIBRARY=%(python_library)s \\
-                -DPYTHON_BASENAME=%(python_basename)s \\
-                -DSETUPTOOLS_DEB_LAYOUT=OFF
+# For more information: https://github.com/ros-melodic-arch/ros-build-tools-py3
+pkgdesc="ROS - %(description)s"
+url='%(site_url)s'
+
+pkgname='ros-%(distro)s-%(arch_package_name)s'
+pkgver='%(package_version)s'
+arch=('any')
+pkgrel=%(package_release)s
+license=('%(license)s')
+
+ros_makedepends=(%(ros_build_dependencies)s)
+makedepends=('cmake' 'ros-build-tools'
+  ${ros_makedepends[@]}
+  %(other_build_dependencies)s)
+
+ros_depends=(%(ros_run_dependencies)s)
+depends=(${ros_depends[@]}
+  %(other_run_dependencies)s)
+
+_dir=%(tarball_dir)s
+source=("${pkgname}-${pkgver}.tar.gz"::"%(tarball_url)s")
+sha256sums=('%(tarball_sha)s')
+
+build() {
+    # Use ROS environment variables
+    source /usr/share/ros-build-tools/clear-ros-env.sh
+    [ -f /opt/ros/%(distro)s/setup.bash ] && source /opt/ros/%(distro)s/setup.bash
+
+    # Create build directory
+    [ -d ${srcdir}/build ] || mkdir ${srcdir}/build
+    cd ${srcdir}/build
+
+    # Fix Python2/Python3 conflicts
+    /usr/share/ros-build-tools/fix-python-scripts.sh -v %(python_version_major)s ${srcdir}/${_dir}
+
+    # Build project
+    cmake ${srcdir}/${_dir} \\
+            -DCMAKE_BUILD_TYPE=Release \\
+            -DCATKIN_BUILD_BINARY_PACKAGE=%(binary)s \\
+            -DCMAKE_INSTALL_PREFIX=/opt/ros/%(distro)s \\
+            -DPYTHON_EXECUTABLE=%(python_executable)s \\
+            -DPYTHON_INCLUDE_DIR=%(python_include_dir)s \\
+            -DPYTHON_LIBRARY=%(python_library)s \\
+            -DPYTHON_BASENAME=%(python_basename)s \\
```

Comment:
I believe these three lines are deprecated and not needed anymore, See the PKGBUILDs in the rest of the organization as confirmation. Basically ROS packages are not using this cmake variable anymore.